docs/amber_script.md

```diff
@@ -156,6 +156,10 @@ BUFFER {name} DATA_TYPE {type} {STD140 | STD430} WIDTH {w} HEIGHT {h} \
 # The buffer will be sized based on the `RENDER_SIZE` of the `PIPELINE`.
 BUFFER {name} FORMAT {format_string} \
     [ MIP_LEVELS _mip_levels_ (default 1) ]
+
+# Load buffer data from a PNG image with file name specified by `FILE`.
```

Comment:
Can you please comment what the file path is? Is it relative to the amber file? Something else? Should I be using `/` or `\` between path elements?

Reply:
The path string is cosumed by LodePNG. / is working on Linux, but I haven't tried this on Windows (yet).